tests/periph_qdec/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       Hello World application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @author      Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "periph/qdec.h"
+#include "xtimer.h"
+
+#define QDEC_ID 0
```

Comment:
Would it be possible to configure all Qdecs ?
Like for [PWM](https://github.com/RIOT-OS/RIOT/blob/master/tests/periph_pwm/main.c) ?

Reply:
sure.